frontend/components/EnrollSecrets/EnrollSecretTable/EnrollSecretRow/EnrollSecretRow.tests.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+import { renderWithSetup } from "test/testingUtils";
+
+import EnrollSecretRow from "./EnrollSecretRow";
+
+const TEAM_SECRET = {
+  secret: "super-secret-secret",
+  created_at: "",
+  team_id: 2,
+};
+describe("Enroll secret row", () => {
+  it("Hides secret by default and shows secret on click of eye icon", async () => {
+    const { user, debug, container } = renderWithSetup(
+      <EnrollSecretRow secret={TEAM_SECRET} />
+    );
+
+    // Secret hidden by default
+    const secretHidden = container.querySelector("input");
+    expect(secretHidden?.type === "password").toBeTruthy();
+
+    // Click eye icon
+    const eyeIcon = screen.findByTestId("eye-icon");
+    await user.click(await eyeIcon);
```

Comment:
do you need the await in `await eyeIcon`?

Reply:
Thank you! I was wondering why it wasn't working, merp, the diff between find by test id and get by test id.